.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Continuous Integration
```

Comment:
If unfamiliar with the Github Actions Workflow syntax please refer to the [docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions) :)